### DISCUSSION TITLE
What is the best way to include typst appendices in a manuscript?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#11590] [Q&A] [answered]
### Description

I have a manuscript project where I want to have appendices which are themselves other manuscripts (with different template).  I am using Typst for pdf.   

How do I do that? Is there any example?  

All the documentation I find on the subject of appendices more or less assumes the project is a book.  Is there a way to create a different kind of project (my kind of manuscript) with appendices?

**Comment by mcanouil** [comment, marked as answer]
Note that there is `.appendix` which is not limited to Book project: https://quarto.org/docs/authoring/appendices.html

But this has currently no meaning in Typst template.

You can have your own Typst template which has an appendix feature via a Lua filter.

See for example:
- https://github.com/typst/typst/discussions/3630


FYI, Typst support will increase as the language grows and its support in Pandoc.
@gordonwoodhull is the one doing most if not all the work in that area for Quarto. This includes support of Book via Typst in the future.
